### DISCUSSION TITLE
Verification process

### DISCUSSION THREAD

**Discussion by drnetcoding** [DesconBelgrade/Klimerko#45] [Q&A]
Hi everyone,

I am interested in the verification process. I couldn't find out how to do it. Also assume that only verified devices can be found in the AirCare application as well?

And another question:
If we share data with the other two communities, where can find the values from the device?

![Capture](https://user-images.githubusercontent.com/78601190/107117145-7d26ef80-6878-11eb-8594-cebba5ac09cd.PNG)

Thanks,
Dusan

**Comment by StevanZS** [comment]
> Hi everyone,
> 
> I am interested in the verification process. I couldn't find out how to do it. Also assume that only verified devices can be found in the AirCare application as well?
> 


Hey @drnetcoding ,

To add your Klimerko sensor to the **AirCare** app, you need to add it to "Vazduh gradjanima" public grounds and contact the AirCare dev by email, send him the device ID and ask him to add it to the app. He's very quick to respond and act, so your sensor should become visible in the app within an hour.

This is a first-hand experience, as I went through this process a few days ago.

Have a great day,
CC
